apps/web/src/store/wallet/hooks.ts

```diff
@@ -36,7 +35,20 @@ import { useAllTokens } from '../../hooks/Tokens';
 import { changeBalances, changeICONBalances } from './reducer';
 
 export function useCrossChainWalletBalances(): AppState['wallet'] {
-  return useSelector((state: AppState) => state.wallet);
+  const signedInWallets = useSignedInWallets();
```

Comment:
why did you update the function?

Reply:
when you signed off of a single chain, wallet would still show the balances of that chain. This way the wallet only shows balances for the chains user is signed in from.